code/modules/reagents/Chemistry-Reagents.dm

```diff
@@ -3831,15 +3823,6 @@
 	description = "Popular with the sailors. Not very popular with everyone else."
 	color = "#664300" //rgb: 102, 67, 0
 
-/datum/reagent/ethanol/deadrum/on_mob_life(var/mob/living/M)
-
-	if(..()) return 1
-
-	M.dizziness += 5
```

Comment:
it makes deadrum and children of deadrum cause additional drunkness though I didn't really mean to remove all of this, I just threw it together quickly.

Drunkness is in /datum/reagent/ethanol